{"uuid": "a7198160-4821-4a46-a132-ad57e1a8c6cb", "children": ["e900d169-1dc5-45f1-8d47-db42eaea0b94", "b691e745-6bac-4211-9100-bf475cb71d78", "b4cd09bb-fef1-4dbf-a29c-83173b0f53ff", "43afab0a-4a2f-4eb4-8235-96adf09ae67b", "68d6fef5-8f3a-4aa6-9c36-12480a290dbc", "4061e1eb-2de6-450c-8100-85ed912eeac0"], "befores": [{"name": "setup_profiles", "status": "failed", "statusDetails": {"message": "Failed: AP latest and active are different\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/wlan-testing/tests/e2e/basic/conftest.py\", line 78, in setup_profiles\n    return_var = fixtures_ver.setup_profiles(request, param, setup_controller, testbed, get_equipment_ref,\n  File \"/wlan-testing/tests/fixtures_2x.py\", line 590, in setup_profiles\n    pytest.fail(\"AP latest and active are different\")\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/outcomes.py\", line 153, in fail\n    raise Failed(msg=msg, pytrace=pytrace)\n"}, "attachments": [{"name": "Active Config: ", "source": "982cbf29-67d9-46d6-be36-86ab6fb7faca-attachment.attach"}, {"name": "Latest Config: ", "source": "b94914f3-8dc2-4a7b-b5d3-1be077d360ac-attachment.attach"}, {"name": "FAILURE: AP Logs: ", "source": "3fe5d467-c768-4d9a-b295-1f8655dc9b03-attachment.attach"}], "start": 1639473737073, "stop": 1639474076040}], "start": 1639473737073, "stop": 1639474077877}